cv.typ

```diff
@@ -377,7 +377,7 @@
     place(
         bottom + right,
         block[
-            #set text(size: 5pt, font: "Consolas", fill: silver)
+            #set text(size: 5pt, font: "Deja Vu Sans Mono", fill: silver)
```

Comment:
Please kindly change to `DejaVu Sans Mono` exactly if this is the proper font name (just basing on what is written in the [Typst Docs](https://typst.app/docs/reference/text/text/#parameters-font))

Reply:
There are inconsistencies in typst repo so thank you for pointing that 😅

![image](https://github.com/user-attachments/assets/83e7b2f4-0c70-42d8-8ec7-4196cd51d7c9)